install_config/configuring_gce.adoc

```diff
@@ -10,183 +10,63 @@
-you want multizone support, edit the *_/etc/origin/cloudprovider/gce.conf_* file as
-shown in xref:configuring-gce-multizone[Configuring Multizone Support in a GCE Deployment].
+include::install_config/topics/configuring_for_gce-proc.adoc[leveloffset=+2]
 
 [[gce-configuring-masters-manual]]
-=== Manually Configuring {product-title} Masters for GCE
-
-To configure the {product-title} masters for GCE:
-
-. Edit or
-xref:../install_config/master_node_configuration.adoc#creating-new-configuration-files[create] the
-master configuration file (*_/etc/origin/master/master-config.yaml_* by default) on all masters and update the
-contents of the `*apiServerArguments*` and `*controllerArguments*` sections:
-+
-[source,yaml]
-----
-kubernetesMasterConfig:
-  ...
-  apiServerArguments:
-    cloud-provider:
-      - "gce"
-    cloud-config:
-      - "/etc/origin/cloudprovider/gce.conf"
-  controllerArguments:
-    cloud-provider:
-      - "gce"
-    cloud-config:
-      - "/etc/origin/cloudprovider/gce.conf"
-----
-+
-[IMPORTANT]
-====
-When triggering a containerized installation, only the directories of
-*_/etc/origin_* and *_/var/lib/origin_* are mounted to the master and node
-container. Therefore, *_master-config.yaml_* should be in *_/etc/origin/master_*
-instead of *_/etc/_*.
-====
-
-. Start or restart the {product-title} services:
-+
-----
-# master-restart api
-# master-restart controllers
-----
 
-[[gce-configuring-nodes]]
-== Configuring Nodes
+include::install_config/topics/manually_configuring_for_gce-proc.adoc[leveloffset=+2]
 
-To configure the {product-title} nodes for GCE:
+[[configuring-gce-registry]]
 
-. Edit the appropriate xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration map] 
-and update the contents of the `*kubeletArguments*` section:
-+
-====
-[source,yaml]
-----
-kubeletArguments:
-  cloud-provider:
-    - "gce"
-  cloud-config:
-    - "/etc/origin/cloudprovider/gce.conf"
-
-----
-====
+include::install_config/topics/configuring_for_gce_registry-proc.adoc[leveloffset=+2]
 
-Currently, the `nodeName` *must* match the instance name in GCE in order
-for the cloud provider integration to work properly. The name must also be
-RFC1123 compliant.
-
-[IMPORTANT]
-====
-When triggering a containerized installation, only the directories of
-*_/etc/origin_* and *_/var/lib/origin_* are mounted to the master and node
-container. Therefore, *_node-config.yaml_* should be in *_/etc/origin/node_*
-instead of *_/etc/_*.
-====
-
-. Start or restart the {product-title} services all nodes.
-+
-ifdef::openshift-enterprise[]
-----
-# systemctl restart atomic-openshift-node
-----
-endif::[]
-ifdef::openshift-origin[]
-----
-# systemctl restart origin-node
-----
-endif::[]
+[[configuring-gce-storage]]
 
+include::install_config/topics/configuring_for_gce_storage-proc.adoc[leveloffset=+2]
 
 [[configuring-gce-multizone]]
-
 == Configuring Multizone Support in a GCE Deployment
```

Comment:
With the new content to this topic being added, I don't want this existing section to get lost. Does it belong somewhere else? Does it tie into one of the new topics? Obviously it's optional, but is it recommended?

Reply:
Can it be configured after install?

Reply:
@e-minguez Sure. I like the info that's here actually, so I've split it into "using Ansible" and "manually doing it" and put it into the sections you mention above. Let me know if you disagree.